data-structures-linked-lists/get-tail.js

```diff
@@ -0,0 +1,6 @@
+export default function getTail(list) {
+  while (list.next !== null) {
```

Comment:
In this function you don't seem concerned about modifying the `list` parameter. ;-)